RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -725,7 +732,8 @@ DAClusterizerInZT_vect::vertices(const vector<reco::TransientTrack> & tracks, co
   // eliminate insigificant vertices, this is more restrictive at higher T
   while (purge(y, tks, rho0, beta)) {
     niter = 0;
-    while ((update(beta, tks, y, true, rho0) > 1.e-6) && (niter++ < maxIterations_)) {}
+    while (( update(beta, tks, y, true, rho0) >  2.5e-7 * y.GetSize() ) && (niter++ < maxIterations_)) {
```

Comment:
perhaps tune up here and in other parts of the last commit can be applied to the non-vectorized clusteriser and it will speed it up already to the convenient enough speed.